modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -3,26 +3,46 @@
 // installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
 
 :_content-type: PROCEDURE
-[id="configuring-host-network-interfaces-in-the-install-config-yaml-file_{context}"]
-= (Optional) Configuring host network interfaces
+[id="configuring-host-network-interfaces-in-the-install-config.yaml-file_{context}"]
+= Optional: Configuring host network interfaces in the `install-config.yaml` file
 
-During installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState. To use the `networkConfig` configuration setting, you must provide an NMState YAML configuration.
+During installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState.
 
 [IMPORTANT]
 ====
-Before configuring host network interfaces, review the OpenShift Container Platform 4.10 release notes.
+Host network interface configuration is set during the installation and provisioning process and includes advanced options such as setting static IPs per host. Before configuring host network interfaces, review the {product-title} 4.10 release notes.
 ====
 
-See link:https://nmstate.io/examples.html#interfaces-ethernet[NMState] for additional examples of the NMState syntax.
+.Prerequisites
 
-.Example
-[source,yaml]
+* Configure a `PTR` DNS record with a valid hostname for each node with a static IP address.
+
+* Install the `jq` package.
+
+.Procedure
+
+. Save the valid `networkConfig` YAML syntax to a file.
+. Generate the configuration file for the `networkConfig` YAML file. Run the following command:
```

Comment:
The command below does not generate the configuration file. Rather, it will tell the user if the nmstate configuration they have created is syntactically correct.

Reply:
So this is the practical application of the option offered in point 2